### PR TITLE
WIP: BL-5658 sms gov notify cleaner and report

### PR DESCRIPTION
**Pull request by raffcecco**

### CODE REVIEW

File: cwl_email_cleaner.tf [hunk 1]
```diff
@@ -52,6 +52,48 @@ resource "aws_cloudwatch_log_metric_filter" "MotrBouncingEmailCleanerPermanentFa
   depends_on = ["aws_cloudwatch_log_group.MotrBouncingEmailCleaner"]
 }
 
+resource "aws_cloudwatch_log_metric_filter" "MotrBouncingEmailCleanerSmsTemporaryFailures_log_metric_filter" {
```
Comment (sebolabs) [inline cwl_email_cleaner.tf]: drop '_log_metric_filter', it's your resource type, there's no need to repeat that in the name

File: cwl_email_cleaner.tf [hunk 2]
```diff
@@ -52,6 +52,48 @@ resource "aws_cloudwatch_log_metric_filter" "MotrBouncingEmailCleanerPermanentFa
...
+  name           = "MotrBouncingEmailCleanerSmsTemporaryFailures_log_metric_filter"
```
Comment (sebolabs) [inline cwl_email_cleaner.tf]: same here

File: cwl_email_cleaner.tf (hunk 1 above)
Comment (raffcecco) [reply cwl_email_cleaner.tf]: Done.

File: cwl_email_cleaner.tf (hunk 2 above)
Comment (raffcecco) [reply cwl_email_cleaner.tf]: Done.

**Pull request merged**